### ISSUE TITLE
influence of mini-batch size (max-tokens per GPU) in wav2vec 2.0

### ISSUE THREAD

**Issue by nastyZhao** [pytorch/fairseq#3869]
#### What is your question?

I tried to run wav2vec 2.0 model with fbank feature instead of wav samples, we used to reached pretrain loss at around 1.8 on 3090 24g GPU, the max-tokens were set to 17500, but when we try to use the same code to train model on 1080 and 2080 card,the max-tokens per GPU were set to 6000 to 9500, after 100 epochs the pretrain loss only reached around 3 and it looks close to converging. 

![Figure_1](https://user-images.githubusercontent.com/16387980/133069526-e027dda4-62e8-43d5-b9ac-4e2a41bb7f23.png)

loss recorded  by different max-tokens setting, 
blue line is max-tokens=2000, lr=0.0005 ;
orange line is max-tokens=6000, lr=0.0005;
green line is max-tokens=9500, lr=0.0001
one token = fbank vector of  four frames
update_freq were set to 2/3 to reach total batch size at around 70000 tokens

#### What have you tried?
So we tried to adjust the max-tokens per GPU and find that larger mini-batch size can achieve lower losses, is that also occurs in normal wav2vec 2.0 training? If so, how to improve the performance with low-memory-GPU?

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master): 1.0.0a0+16c1a20 
 - PyTorch Version (e.g., 1.0): 1.9
 - OS (e.g., Linux): Linux
 - How you installed fairseq (`pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.8
 - CUDA/cuDNN version: 11.1
 - GPU models and configuration: 4 GPU 2080 10Gb memory 
 - Any other relevant information:
